extensions/tekton/generator/cmd/generate/generate.go

```diff
@@ -16,43 +16,65 @@
-	ClusterTask                     v1alpha1.ClusterTask
-	ClusterTaskList                 v1alpha1.ClusterTaskList
-	Condition                       v1alpha1.Condition
-	ConditionList                   v1alpha1.ConditionList
+  V1alpha1Condition                       v1alpha1.Condition
+  V1alpha1ConditionList                   v1alpha1.ConditionList
+  V1alpha1PipelineResource                v1alpha1.PipelineResource
+  V1alpha1PipelineResourceList            v1alpha1.PipelineResourceList
+
+  V1beta1Pipeline                        v1beta1.Pipeline
+  V1beta1PipelineList                    v1beta1.PipelineList
+  V1beta1PipelineRun                     v1beta1.PipelineRun
+  V1beta1PipelineRunList                 v1beta1.PipelineRunList
+  V1beta1PipelineRunTaskRunStatus        v1beta1.PipelineRunTaskRunStatus
+  V1beta1PipelineRunConditionCheckStatus v1beta1.PipelineRunConditionCheckStatus
+  V1beta1Task                            v1beta1.Task
+  V1beta1TaskList                        v1beta1.TaskList
+  V1beta1TaskRun                         v1beta1.TaskRun
+  V1beta1TaskRef                         v1beta1.TaskRef
+  V1beta1TaskRunList                     v1beta1.TaskRunList
+  V1beta1ClusterTask                     v1beta1.ClusterTask
+  V1beta1ClusterTaskList                 v1beta1.ClusterTaskList
+  V1beta1PipelineTask                    v1beta1.PipelineTask
+  V1beta1PipelineTaskResource            v1beta1.PipelineTaskResources
+  V1beta1ResourceDeclaration             v1beta1.ResourceDeclaration
+	V1beta1Sidecar                         v1beta1.Sidecar
+	V1beta1SidecarState                    v1beta1.SidecarState
+	V1beta1PodTemplate                     v1beta1.PodTemplate
+	V1beta1Step                            v1beta1.Step
+	V1beta1TaskResource                    v1beta1.TaskResource
+	V1beta1TaskResourceBinding             v1beta1.TaskResourceBinding
+
+	Template                               pod.Template
+	ResourceDeclaration                    resource.ResourceDeclaration
+	resource                               resource.PipelineResource
+
 }
 
 func main() {
 	packages := []schemagen.PackageDescriptor{
 		//"$ref": "#/definitions/github_com_tektoncd_pipeline_vendor_k8s_io_api_core_v1_Probe",
 		{"github.com/tektoncd/pipeline/pkg/apis/pipeline/v1alpha1", "tekton.dev", "io.fabric8.tekton.pipeline.v1alpha1", "tekton_v1alpha1_"},
-		{"github.com/knative/pkg/apis", "v1", "io.fabric8.knative.v1", "knative_"},
+    {"github.com/tektoncd/pipeline/pkg/apis/pipeline/v1beta1", "tekton.dev", "io.fabric8.tekton.pipeline.v1beta1", "tekton_v1alpha2_"},
+    {"github.com/tektoncd/pipeline/pkg/apis/resource/v1alpha1", "tekton.dev", "io.fabric8.tekton.resource.v1alpha1", "tekton_resource_v1alpha1_"},
+    {"github.com/tektoncd/pipeline/pkg/apis/pipeline/pod", "tekton.dev", "io.fabric8.tekton.pipeline.pod", "tekton_pod_"},
+    {"github.com/knative/pkg/apis", "v1", "io.fabric8.knative.v1", "knative_"},
```

Comment:
Why do we need?

Reply:
Umm, not sure. I didn't touch it while upgrading model. Let me try removing it.

Reply:
Looks like some structs in tekton model depend on these packages(one of these is this https://github.com/tektoncd/pipeline/blob/master/pkg/apis/pipeline/v1beta1/pipelinerun_types.go#L28 ), when I removed them I started getting errors.

Reply:
ahh, no worries